modules/intro/pages/applications.adoc

```diff
@@ -40,10 +40,63 @@ Duplicating an application is useful for quickly trying out new options while st
 
 === Share an application
 
+==== Share via login
 Click on btn:[Share] in the three-dot menu to bring up the Share window.
 
 image::share-application.png[]
 
+==== Share via access token
+
+Access tokens may also be used to share application widgets with other users without a login for insights.
+
+===== Creating database account
+
+You will need a separate database account to generate an access token for Insights.
+Ensure that you grant minimal privileges to this account.
+
+Please check the xref:tigergraph-server:reference:list-of-privileges.adoc[].
+
+TigerGraph provides two types of queries: **Installed** and **Interpret**.
+
+The **Interpret** query is the default query type, unless you explicitly convert it to an **Installed** query.
+
+==== Write query permission is required to run interpret queries.
+
+`READ_DATA` privilege is required for any type of query, and you can grant it at various levels.
+(Details are provided in xref:gui:admin-portal:management/user-management.adoc[local privileges])
+
+`READ_SCHEMA` and `READ_QUERY` are also needed for any type of query.
+
+.The minimal required privileges for different types of queries:
+[cols='2']
+|===
+| Query Type | Minimal Required Privileges
+
+| Installed Query | `READ_DATA` ,`READ_SCHEMA`, `READ_QUERY`
+| Interpret Query | `READ_DATA` ,`READ_SCHEMA` ,`READ_QUERY`, `WRITE_QUERY`
+|===
+
+==== (Optional) Config nginx
+
+This is only needed if you want to embed insights into third party webpages.
+
+Go to admin portal to config `nginx`.
+
+[source, console]
+Content-Security-Policy: frame-ancestors 'self' https://www.thirdparty.com;
+
+==== Generating access token
+
+Use the account you just created to generate an access token.
+
+Go to `Application > Settings > Security` to create the access token.
+
+At the security page, we also provide a standalone application URL based on the access token.
```

Comment:
As a user, I don't understand what to do. 

1. Is this the Application page of the Admin Portal or Insights? (I _think_ we want the user to switch back from Admin Portal to Insights. That needs to be said explicitly.)
2. Once I am at the Security page, what do I do?
3. Why are you mentioning "also provide a ... URL"? Why do I care? What can/should I do with this URL?

Reply:
Asked the developer in the ticket. updated in latest commit based on their answers.